src/App/App.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {
+  BrowserRouter,
+  Route,
+  Switch
+} from 'react-router-dom';
+
+import List from '../UserManagement/List';
+import Header from '../Header';
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Header />
+      <Switch>
```

Comment:
Hier ist der Switch der entscheidet, welche Komponente gerendert werden soll. Hier wird z.B. bei `<List />` gerendert, wenn der Pfad gerade `/users` ist. Ändern lässt sich der Pfad mit den `<Link />` Komponenten von React-Router. Die kommen weiter unten in `GameLink` zum Einsatz.